devtools/maven/src/main/java/io/quarkus/maven/QuarkusProjectMojoBase.java

```diff
@@ -59,13 +59,13 @@ public abstract class QuarkusProjectMojoBase extends AbstractMojo {
     protected List<RemoteRepository> repos;
 
     @Parameter(property = "bomGroupId", required = false)
-    private String bomGroupId;
+    protected String bomGroupId;
 
     @Parameter(property = "bomArtifactId", required = false)
-    private String bomArtifactId;
+    protected String bomArtifactId;
 
     @Parameter(property = "bomVersion", required = false)
-    private String bomVersion;
+    protected String bomVersion;
```

Comment:
Thinking more about it, perhaps, these parameters should be reserved for a case when the mojo is executed outside a project.
If we are in a context of a project, the BOMs should probably be read from the project. If a user wants a different version than what's found in the project, they should be able to set a quarkus version property in the project. For example, in the projects we generate it could be `-Dquarkus.platform.version=XXX`, etc. Otherwise, we are "disconnecting" from the project config with `bomVersion` and other `bomXXX` parameters.

Does it make sense?